Index: skills-lock.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n  \"version\": 1,\r\n  \"skills\": {\r\n    \"supabase\": {\r\n      \"source\": \"supabase/agent-skills\",\r\n      \"sourceType\": \"github\",\r\n      \"skillPath\": \"skills/supabase/SKILL.md\",\r\n      \"computedHash\": \"61638e85394d2e39d1109cbf607593afb9733e0de19cdf0b52ec9bc32d95ea74\"\r\n    },\r\n    \"supabase-postgres-best-practices\": {\r\n      \"source\": \"supabase/agent-skills\",\r\n      \"sourceType\": \"github\",\r\n      \"skillPath\": \"skills/supabase-postgres-best-practices/SKILL.md\",\r\n      \"computedHash\": \"3639bed1f40b3fbadae79fee631c42c89b2d1f5c30b05d5aa3cca06422a6bbbc\"\r\n    }\r\n  }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/skills-lock.json b/skills-lock.json
--- a/skills-lock.json	(revision 5bc24f7943b3dcff1203b2fee8be663f4bc656c9)
+++ b/skills-lock.json	(date 1784794804575)
@@ -1,6 +1,18 @@
 {
   "version": 1,
   "skills": {
+    "migrate-radix-to-base": {
+      "source": "shadcn/ui",
+      "sourceType": "github",
+      "skillPath": "skills/migrate-radix-to-base/SKILL.md",
+      "computedHash": "4fdc31bf287f674ba6bf91a648120f7b34cd7ef370f998f2cc116fed067789c5"
+    },
+    "shadcn": {
+      "source": "shadcn/ui",
+      "sourceType": "github",
+      "skillPath": "skills/shadcn/SKILL.md",
+      "computedHash": "4f78ff7cd3a4f637b6fe30dbbab4a80a19dd63fa62d9352bed461ccfdbcbbf43"
+    },
     "supabase": {
       "source": "supabase/agent-skills",
       "sourceType": "github",
